core/frontend/src/tile/OrbitGtTileTree.ts

```diff
@@ -388,17 +395,18 @@ export namespace OrbitGtTileTree {
 /** Supplies a reality data [[TileTree]] from a URL. May be associated with a persistent [[GeometricModelState]], or attached at run-time via a [[ContextOrbitGtState]].
  * @internal
  */
-class OrbitGtTreeReference extends RealityModelTileTree.Reference {
+export class OrbitGtTreeReference extends RealityModelTileTree.Reference {

```

Comment:
After you change `instanceof` check, this will no longer need to be exported. Neither will `RealityTreeReference`.

Reply:
OK, Cool.  Good catch.